### ISSUE TITLE
Ay & sid support

### ISSUE THREAD

**Issue by redbug26** [jhe2/gmu#9]
It will be great if gmu could support ay module (msx/atarist/amstrad/...) & sid (c64).
Do you think that you could add it ?

If not, do you accept pull request ? 

Modizer already use such librairies that could be a great help: https://github.com/yoyofr/modizer

Thanks

**Comment by jhe2** [comment]
I agree, supporting more module formats and SID would be nice. Given there are decoder libraries for the desired formats available, implementing decoder plugins for Gmu is fairly straight forward.
I could add support for those formats if there are usable decoder libraries, could take me little while to get around to do that though.

Of course, I would also accept pull requests. So if you would like to give it a go, you are welcome to do so. Please keep in mind to use libraries with open source licenses that are compatible with Gmu's GPLv2 license and vice versa (e.g. MIT, BSD, Apache, LGPL or GPLv2+). The library you've been linking to looks interesting, but I wasn't able to figure out its licensing terms at a glance.
For SID I know there is ``libsidplayfp`` (which is a fork of the older ``libsidplay2``). It is GPLv2 licensed, so there is no problem using that.

**Comment by redbug26** [comment]
Yes, libsidplayfp is also available in my ubuntu ;) 
For .ay/.ym, there is also player available in opencubicplayer at https://github.com/mywave82/opencubicplayer/tree/master/playay and it's GPLv2.

I've compiled gmu on my linux and it works great also on true desktop. (it will be great if it was available in the ubuntu repository :p)
Thanks again for your work.
